docs/security/05-05-production-profile.md

```diff
@@ -5,35 +5,105 @@ type: Configuration
+
+#### Internal PostgreSQL setup
 The database is created in the cluster as a StatefulSet and uses a PersistentVolume that is provider-specific. This means that the PersistentVolume used by the database uses the default StorageClass of the cluster's host provider.
 
-Alternatively, you can use a compatible external database to store the registered client data. To use an external database, you must create a Kubernetes Secret with the database password in the same Namespace as the Hydra OAuth2 server. The details of the external database are passed through these parameters of the production profile override:
+In order to use the database, the following configuration parameters have to be supplied:
+
+**General settings:**
+
+| Parameter | Description | Required value |
+|----------|-----------| :---: |
+| **global.ory.hydra.persistence.enabled** | Defines whether Hydra should use the `database` mode of operation. If `false`, it uses the `in-memory` mode. | `true` |
+| **global.ory.hydra.persistence.postgresql.enabled** | Defines whether Hydra should initiate the deployment of an in-cluster database. Set to `false` to use a self-provided database. If set to `true`, Hydra always uses an in-cluster database and ignores the custom database details. | `true` |
+| **hydra.hydra.autoMigrate** | Enables Hydra auto-migration feature, which prepares the database | `true` | 
+
+You can find an example of the required [configmap here](./assets/003-ory-db-postgres.yaml)
+
+#### Custom database
+
+Alternatively, you can use a compatible, custom database to store the registered client data. To use a database, you must create a Kubernetes Secret with the database password as an override for the Hydra OAuth2 server. The details of the database are passed using these parameters of the production profile override:
+
+**General settings:**
+
+| Parameter | Description | Required value |
+|----------|------| :---: |
+| **global.ory.hydra.persistence.enabled** | Defines whether Hydra should use the `database` mode of operation. If false, it uses the `in-memory` mode. | `true` |
+| **global.ory.hydra.persistence.postgresql.enabled** | Defines whether Hydra should initiate the deployment of an in-cluster database. Set to `false` to use a self-provided database. If set to `true`, Hydra always uses an in-cluster database and ignores the custom database details. | `false` |
+| **hydra.hydra.autoMigrate** | Enables Hydra auto-migration feature, which prepares the database. | `true` | 
+| **hydra.hydra.config.secrets.system** | Sets the system encryption string for Hydra. | An at least 16 characters long alphanumerical string | 
+| **hydra.hydra.config.secrets.cookie** | Sets the cookie session encryption string for Hydra. | An at least 16 characters long alphanumerical string |
+
+**Database settings:**
+
```

Comment:
Currently, the document is weirdly formatted, take a look : https://deploy-preview-7130--kyma-project-docs-preview.netlify.com/components/security/#configuration-o-auth2-server-production-profile-google-cloud-sql